src/main/java/mate/academy/Main.java

```diff
@@ -1,11 +1,14 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
 import mate.academy.model.Movie;
 import mate.academy.service.MovieService;
+import mate.academy.service.impl.MovieServiceImpl;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
+        MovieService movieService = (MovieServiceImpl) injector.getInstance(MovieService.class);
```

Comment:
Casting the `MovieService` to `MovieServiceImpl` is redundant and can lead to a ClassCastException if the implementation changes. You should use the interface type directly when declaring the `movieService` variable.